tests/messages.cpp

```diff
@@ -858,7 +858,7 @@ TEST(RoomEvents, CallAnswer)
               "sdp": "v=0\r\no=- 6584580628695956864 2 IN IP4 127.0.0.1[...]",
               "type": "answer"
             },
-            "version": 0
+            "version": "0"
```

Comment:
I don't think you should change these, add tests for v1 and for v0 with the integer.